From 76b33453c4da51ce6636ef0bc2667a94b20a2c5d Mon Sep 17 00:00:00 2001
From: LuK1337 <priv.luk@gmail.com>
Date: Thu, 20 Sep 2018 10:39:49 +0200
Subject: [PATCH 18/19] Settings: Improve phone number preference ordering

* Make sure there's enough space to fit all SIM slots.

Change-Id: I63e447303e1b5ce222132b196b64fcc2fa28b799
---
 res/xml/my_device_info.xml | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/res/xml/my_device_info.xml b/res/xml/my_device_info.xml
index 606cada038..381c1ca8cf 100644
--- a/res/xml/my_device_info.xml
+++ b/res/xml/my_device_info.xml
@@ -51,27 +51,27 @@
 
     <Preference
         android:key="emergency_info"
-        android:order="4"
+        android:order="10"
         android:title="@string/emergency_info_title"
         android:summary="@string/summary_placeholder"/>
 
     <!-- Legal information -->
     <Preference
         android:key="legal_container"
-        android:order="5"
+        android:order="11"
         android:title="@string/legal_information"
         android:fragment="com.android.settings.LegalSettings"
         settings:allowDividerAbove="true"/>
 
     <!-- Contributors cloud -->
     <org.lineageos.internal.lineageparts.LineagePartsPreference
-        android:order="6"
+        android:order="12"
         android:key="contributors"/>
 
     <!-- Regulatory labels -->
     <Preference
         android:key="regulatory_info"
-        android:order="6"
+        android:order="13"
         android:title="@string/regulatory_labels">
         <intent android:action="android.settings.SHOW_REGULATORY_INFO"/>
     </Preference>
@@ -79,7 +79,7 @@
     <!-- SIM status -->
     <Preference
         android:key="sim_status"
-        android:order="7"
+        android:order="14"
         android:title="@string/sim_status_title"
         settings:keywords="@string/keywords_sim_status"
         android:summary="@string/summary_placeholder"
-- 
2.17.1

